static/js/blame.js

```diff
@@ -194,6 +194,11 @@ var BlameStripHoverHandler = new (class BlameStripHoverHandler {
       // Don't care about clicks outside the blame strip if there's no popup showing.
       return;
     }
+    if (clickedOutsideBlameStrip && event.target.closest("#blame-popup")) {
```

Comment:
Maybe change the `.closest` call to be `BlamePopup.popup.contains(event.target)`?

Reply:
Ah, thanks. I was looking for something like that but I was looking through `Element` functions on MDN and `contains` is a `Node` function so I didn't see it...